Robert Stoelhorst/week01/day_04/functions/homework.js

```diff
@@ -0,0 +1,69 @@
+// Geometry Function Lab
+// Part 1, Rectangle
+// Given the following a rectangle object like the one below, write the following functions which accept a rectangle object as an argument:
+//
+// isSquare - Returns whether the rectangle is a square or not
+// area - Returns the area of the rectangle
+// perimeter - Returns the perimeter of the rectangle
+
+const areaOfRectangle = function (x, y) {
+  const area = x * y;
+  const perimeter = x * 2 + y * 2;
+  if (x === y)
+return (` This is a square and the area of this square is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
+  if (x !== y)
+return (` The area of this rectangle is ${ area }cm2 and it's perimeter is ${ perimeter }cm`)
+}
+  areaOfRectangle(10, 10)
```

Comment:
I know this function works, but if I'm being particular about your indentation and syntax it should look like: 

const areaOfRectangle = function (x, y) {
  const area = x * y;
  const perimeter = x * 2 + y * 2;
  if (x === y) {
      return (` This is a square and the area of this square is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
  }
  if (x !== y) {
      return (` The area of this rectangle is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
  }
}